workshop: first commit (#33)
facebookresearch/RAM: workshop/COLM_2025/README.md

# Reasoning, Attention & Memory: RAM 2 – 10 Years On

Ten years ago in Montreal 2015, the RAM workshop took place to bring together the burgeoning field covering the "interplay of reasoning, attention and memory", just before Transformers were invented – but when many of the components to get there had just been published and were in place. The workshop included many speakers who are still prominent in pushing these directions today: Yoshua Bengio, Kyunghyun Cho, Jürgen Schmidhuber, Sainbayar Sukhbaatar, Ilya Sutskever, and more. See the historical website for more details.

Ten years later, we are hosting RAM 2 in the same location in Montreal, with a two-fold purpose. Firstly, as a retrospective and analysis of what has happened in the last 10 years. We are inviting presenters from the first workshop to this end, as well as to add their current perspectives. Hence secondly, and more importantly, we will bring together the field to discuss new trends and future directions for the next 10 years – which is further enabled by inviting new speakers, panelists and poster presenters discussing these fresh ideas.

> Why does this make sense? The RAM topic is as important as ever, and has gone on to dominate the field. These new directions include:

Reviewer: ```suggestion
Why does this make sense? The RAM topic is as important as ever, and has gone on to dominate the field. These new directions include:

```

> (R): New reasoning methods including both token-based and that use continuous vectors, and how they combine with memory.
> (A): New attention methods that enable better reasoning and use of short and long-term memory.
> (M): Architectural changes to LLMs to improve memory and reasoning capabilities.

Reviewer: ```suggestion
(R): New reasoning methods including both token-based and that use continuous vectors, and how they combine with memory.

(A): New attention methods that enable better reasoning and use of short and long-term memory.

(M): Architectural changes to LLMs to improve memory and reasoning capabilities.
```

Overall, we highlight that the workshop is most concerned with methods that aim to explore the interplay between these three aspects.

## Call for Papers

> We will host paper submissions on open review, link will be provided later. We invite researchers and practitioners to submit their work to the COLM 2025 Workshop on Reasoning, Attention & Memory (RAM@COLM25).   

Reviewer: ```suggestion
We will host paper submissions on open review, link will be provided later. We invite researchers and practitioners to submit their work to the COLM 2025 Workshop on Reasoning, Attention & Memory 2 (RAM2@COLM25).   
```

* **Submission Deadline:** June 23, 2025
* **Author Notification Deadline:** July 24, 2025
* **Submission Details:** Submissions should follow [general guide for COLM conference](https://colmweb.org/cfp.html). Papers can be up to 9 pages (not including references) and have to be anonymized.  All submissions must be in PDF format, please use the [LaTeX style files provided by organizers](https://github.com/COLM-org/Template/archive/refs/tags/2025.zip).

## Invited speakers

> + Yoshua Bengio, Univ. of Montreal
> + Kyunghyun Cho, NYU & Prescient Design
> + Yejin Choi, Stanford &  NVIDIA
> + Azalia Mirhoseini, Stanford
> + Sainbayar Sukhbaatar, Meta
> + Jason Wei, OpenAI

Reviewer: ```suggestion
+ Yoshua Bengio, Univ. of Montreal
+ Kyunghyun Cho, NYU & Prescient Design
+ Yejin Choi, Stanford & NVIDIA
+ Azalia Mirhoseini, Stanford
+ Juergen Schmidhuber, KAUST
+ Sainbayar Sukhbaatar, Meta
+ Jason Wei, OpenAI
```

> ## Tentative schedule:
> 8x 30 minutes invited talks 
> 3x 30 minute panel discussions in between groups of talks, aiming to cluster by topic.
> 2 hour poster session with 1 hour optional break for lunch in between

Reviewer: ```suggestion
## Tentative schedule:
TBD
```
